sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -50,4 +57,41 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     program.institution = { id: educationProgram.institutionId } as Institution;
     return this.repo.save(program);
   }
+
+  async getEducationProgramsSummary(
+    institutionId: number,
+  ): Promise<EducationProgramsSummary[]> {
+    const summaryResult = await this.repo
+      .createQueryBuilder("programs")
+      .select([
+        "programs.id as id",
+        "programs.name as name",
+        "programs.cipCode as cipCode",
+        "programs.credentialType as credentialType",
+        "programs.credentialTypeOther as credentialTypeOther",
+        "programs.approvalStatus as approvalStatus",
+      ])
+      .addSelect(
+        (query) =>
+          query
+            .select("COUNT(*)")
+            .from(EducationProgramOffering, "offerings")
+            .where("offerings.educationProgram.id = programs.id"),
```

Comment:
Can we add the locationId also in the condition for the query? As institutionlocation is an onetoone relation to the educationprogramoffering table. Please share your thoughts. I feel, as program is shared across locations, it would be better. Thought??

Reply:
Yes, saw it just after sent the PR. I am testing the change :D

Reply:
Fixed!